net-dns/dnsdist/dnsdist-1.1.0-r2.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Likewise, the commit message should tell what's really happening. Adding a new revision is just a side effect, I don't need the commit message to tell me that.